README_ko.md

```
@@ -118,111 +143,180 @@ sudo cp target/release/openpista /usr/local/bin/
-[database]
 url = "~/.openpista/memory.db"
-
-[skills]
 workspace = "~/.openpista/workspace"
 ```
 
-### 환경 변수
+### 환경 변수 오버라이드 (CI / 스크립트용)
 
+환경 변수는 설정 파일 값을 덮어씁니다. CI 파이프라인, Docker, 비대화형 환경용이며 — 기본 설정 방법이 아닙니다.
 | 변수 | 설명 |
```

markdownlint-cli2 (0.21.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)